### PR TITLE
Remove needless read_rda2()

### PR DESCRIPTION
**Pull request by maurolepore**
This was a temporary function I forgot to remove.

### CODE REVIEW

File: R/read_rda.R [hunk 1]
```diff
@@ -21,13 +21,6 @@
 #' # Fails
 #' try(read_rda(path))
 #' @noRd
-read_rda2 <- function(file_name) {
```
Comment (maurolepore) [inline R/read_rda.R:24]: Sorry, I forgot to remove this temporary function.

File: R/read_rda.R [hunk 2]
```diff
@@ -21,13 +21,6 @@
...
-  tryCatch(
-    read_rdata(file_name),
-    # Unfortunately, legacy code saved .rds files with the extension .rda
-    error = function(e) readr::read_rds(file_name)
-  )
-}
 read_rda <- function(file_name) {
   tryCatch(
     # Unfortunately, legacy code saved .rds files with the extension .rda
```
Comment (maurolepore) [inline R/read_rda.R:26]: This does not have the happy path first :( -- it wasn't passing tests and I can't refactor right now. Anyway, this was flagged as Nit so shoud be good for now.

Comment (maurolepore) [reply R/read_rda.R:26]: Also, I renamed the .R file to reflect the name of the function.

**Review by maurolepore — COMMENTED** [review]
Here are the highlights.

**Review by cjyetman — APPROVED** [review]
LGTM

**Pull request merged**